lib/network/modules/Canvas.js

```diff
@@ -249,8 +249,7 @@ class Canvas {
     this.hammer.on('pinch',     (event) => {this.body.eventListeners.onPinch(event)});
 
     // TODO: neatly cleanup these handlers when re-creating the Canvas, IF these are done with hammer, event.stopPropagation will not work?
-    this.frame.canvas.addEventListener('mousewheel',     (event) => {this.body.eventListeners.onMouseWheel(event)});
-    this.frame.canvas.addEventListener('DOMMouseScroll', (event) => {this.body.eventListeners.onMouseWheel(event)});
+    this.frame.canvas.addEventListener('wheel',     (event) => {this.body.eventListeners.onMouseWheel(event)});
```

Comment:
Just a cosmetic change that really annoys me. Could you leave just one space here, please?

Reply:
Didn't notice it added extra spaces. They're removed.